rastervision_core/rastervision/core/rv_pipeline/semantic_segmentation.py

```diff
@@ -33,9 +36,8 @@ def filter_windows(windows):
             filt_windows = []
             for w in windows:
                 label_arr = label_source.get_labels(w).get_label_arr(w)
-                null_inds = (
-                    label_arr.ravel() == class_config.get_null_class_id())
-                if not np.all(null_inds):
+                null_mask = label_arr == class_config.get_null_class_id()
```

Comment:
Shouldn't we be filtering based on whether or not the image from the raster source has NODATA pixels, rather than based on null classes in the label array? (In other words, shouldn't you do the same thing that you did in object detection and chip classification?

Reply:
It was already filtering based on null labels and not based on NODATA pixels like the other pipelines, so I assumed those were equivalent. But I guess that is not necessarily true.